addons/context_actions/stringtable.xml

```diff
@@ -38,100 +44,130 @@
+            <Chinesesimp>10米</Chinesesimp>
+            <Chinese>10米</Chinese>
         </Key>
         <Key ID="STR_ZEN_Context_Actions_50m">
             <English>50 m</English>
             <German>50 m</German>
             <Korean>50 m</Korean>
             <Japanese>50 m</Japanese>
+            <Chinesesimp>50米</Chinesesimp>
+            <Chinese>50米</Chinese>
         </Key>
         <Key ID="STR_ZEN_Context_Actions_100m">
             <English>100 m</English>
             <German>100 m</German>
             <Korean>100 m</Korean>
             <Japanese>100 m</Japanese>
+            <Chinesesimp>100米</Chinesesimp>
+            <Chinese>100米</Chinese>
         </Key>
         <Key ID="STR_ZEN_Context_Actions_250m">
             <English>250 m</English>
             <German>250 m</German>
             <Korean>250 m</Korean>
             <Japanese>250 m</Japanese>
+            <Chinesesimp>250米</Chinesesimp>
+            <Chinese>250米</Chinese>
         </Key>
         <Key ID="STR_ZEN_Context_Actions_HoldFire">
             <English>Hold Fire</English>
             <German>Feuer halten</German>
             <Korean>사격 중지</Korean>
             <Polish>Wstrzymaj Ogień</Polish>
             <Japanese>発砲禁止</Japanese>
+            <Chinesesimp>停火</Chinesesimp>
+            <Chinese>停火</Chinese>
         </Key>
         <Key ID="STR_ZEN_Context_Actions_HoldFireDefend">
             <English>Hold Fire, Defend Only</English>
             <German>Feuer halten, nur verteidigen</German>
             <Korean>사격 중지, 방어만 하십시오</Korean>
             <Polish>Wstrzymaj ogień, tylko obraniaj</Polish>
             <Japanese>発砲禁止、自衛のみ</Japanese>
+            <Chinesesimp>停火，仅防御</Chinesesimp>
+            <Chinese>停火，僅防禦</Chinese>
         </Key>
         <Key ID="STR_ZEN_Context_Actions_HoldFireEngage">
             <English>Hold Fire, Engage At Will</English>
             <German>Feuer halten, selbstständiger Angriff</German>
             <Korean>사격 중지, 자유 사격 개시</Korean>
             <Polish>Wstrzymaj ogień, Inicjuj samowolnie</Polish>
             <Japanese>発砲禁止、命令時射撃</Japanese>
+            <Chinesesimp>停火，随意反击</Chinesesimp>
+            <Chinese>停火，隨意反擊</Chinese>
         </Key>
         <Key ID="STR_ZEN_Context_Actions_FireAtWill">
             <English>Fire At Will</English>
             <German>Feuerfreigabe</German>
             <Korean>자유 사격 개시</Korean>
             <Polish>Otwórz ogień samowolnie</Polish>
             <Japanese>随意射</Japanese>
+            <Chinesesimp>随意开火</Chinesesimp>
+            <Chinese>隨意開火</Chinese>
         </Key>
         <Key ID="STR_ZEN_Context_Actions_FireAtWillEngage">
             <English>Fire &amp; Engage At Will</English>
             <German>Feuerfreigabe und selbstständiger Angriff</German>
             <Korean>사격 &amp; 자유 사격 개시</Korean>
             <Polish>Ogien i samowolona inicjacja</Polish>
             <Japanese>随意射、命令時射撃</Japanese>
+            <Chinesesimp>随意开火 &amp; 随意反击</Chinesesimp>
```

Comment:
`&amp;` is just the HTML code for an ampersand.

Reply:
Thanks for the reminder, but the question at hand is very confusing to me. It suggests that there are three files with space indentation errors. But everything is correct when I look at the files both in Github and VSCode.

Reply:
If you [enable commits from maintainers](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork), we could just quickly run a script over it to fix it 😉

Reply:
You have tabs instead of spaces in those files. We use 4 spaces per indentation level.

Reply:
I enable allow edits by maintainers. Not sure if it's the same function。
I have once again rewritten the wrong line and the problem should now be solved, hopefully！

Reply:
Yes, it's the correct one, but I see you solved it on your own in the meantime 🥳